test/api/Auth.js

```diff
@@ -189,4 +192,44 @@ describe('Auth', () => {
     });
   });
 
+  describe('checkIfPageCountResets()', () => {
+    it('Should reset user\'s page count when last login was over 7 days ago',
+      async () => {
+        // mock current day to January 10th, 2023 (Tuesday)
+        const mockCurrentDate = mockDayMonthAndYear(10, 0, 2023);
+        // mock last login to January 1st, 2023 (Sunday)
+        const mockLastLogin = new Date(2023, 0, 1);
+        const result = checkIfPageCountResets(mockLastLogin);
+        expect(result).to.be.true;
+      });
+
+    it('Should reset user\'s page count when there\s a Sunday between last login and now',
```

Comment:
another example rename, the other ones should do the same thing

```suggestion
    it('Should reset page count when there is a Sunday between last login and now',
```